pacmod_msgs/msg/SystemCmdInt.msg

```diff
@@ -49,6 +49,10 @@ uint16 TURN_NONE = 1
 uint16 TURN_LEFT = 2
 uint16 TURN_HAZARDS = 3
 
+uint16 REAR_PASS_DOOR_NEUTRAL = 0
+uint16 REAR_PASS_DOOR_OPEN = 1
+uint16 REAR_PASS_DOOR_CLOSE = 2
+
```

Comment:
Sorry, one last change. Since these values would apply to other doors, can you remove the REAR_PASS_ part from the names?